script.js

```diff
@@ -0,0 +1,682 @@
+/* eslint-disable no-use-before-define */
+/* eslint-disable prefer-template */
+/* eslint-disable indent */
+/* eslint-disable space-before-blocks */
+/* eslint-disable max-len */
+/* eslint-disable default-case */
+/* eslint-disable no-extra-semi */
+/* eslint-disable semi */
+/* eslint-disable eol-last */
+/* eslint-disable vars-on-top */
+/* eslint-disable no-var */
+/* eslint-disable space-before-function-paren */
+/* eslint-disable func-names */
+/* eslint-disable padded-blocks */
+/* eslint-disable no-plusplus */
+/* eslint-disable no-undef */
```

Comment:
ESLint enabled: +10
necessary rules disabled: -15
without disabling them there are a lot of violations